js/app/src/api_docs/CodeSnippets.svelte

```diff
@@ -64,19 +64,16 @@ result = client.predict(<!--
 				-->{/if}<!--
 			--><span class="desc"
 								><!--
-			-->	# {python_type.type} {#if python_type.description}({python_type.description})
-								{/if}<!--
-			-->in '{label}' <!--
+			-->	# {python_type.type} {#if python_type.description}({python_type.description}){/if}<!---->in '{label}' <!--
```

Comment:
There should be a space before the "in":

<img width="214" alt="image" src="https://github.com/gradio-app/gradio/assets/1778297/3fb0f281-9f30-4801-870c-cb53fcc79515">

Reply:
oops! missed that